tools/notification-configuration/identity-resolver/identity-resolver.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>identity_resolver</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="2.2.0" />
+    <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.19317.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\github-codeowner-subscriber\github-codeowner-subscriber.csproj" />
```

Comment:
Why not just add the support you need directly to github-codeowner-subscriber instead of creating another tool?

Reply:
Because I would have then been called out for adding non-`github-codeowner-subscriber` functionality to that tool.

Reply:
Granted, that's kinda what I'm doing by re-using this solution anyway though :/